screens/ActionsType.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigation, Link } from '@react-navigation/native';
+import {  TouchableOpacity} from 'react-native';
+import server from '../common/server';
+import { API, COLORS } from '../common/constants';
+import { useAppDispatch } from '../stores/hooks';
+import {Text, View,  Box, Radio, Button, Image, ScrollView } from 'native-base';
+import BackIcon from '../assets/images/back-icon.svg'
+import Checkbox from 'expo-checkbox';
```

Comment:
Why did you use this library component instead of the NativeBase one?

Reply:
I used the expo checkbox because I was getting and error with the native base one.